web/src/narrow.js

```diff
@@ -81,6 +81,14 @@ export function save_narrow(operators) {
     changehash(new_hash);
 }
 
+function set_recent_view(operators) {
+    if (browser_history.state.changing_hash) {
+        return;
+    }
+    const new_hash = hash_util.operators_to_hash(operators, true);
+    changehash(new_hash);
+}
```

Comment:
I think we want to use a different hash structure, like `/recent/stream/{stream_id}`, for the recent view. So I think there' s no need for this code to be here or to call `changehash`?

Reply:
Oh yes, so I think this code will be needed for when we use this view in other ways (e.g. clicking on the stream name in the left sidebar), but not for this PR. Good point.